### PR TITLE
Added nutrition concepts

### PR DESCRIPTION
**Pull request by reagan-meant**

### CODE REVIEW

File: configuration/concepts/misc.csv [hunk 1]
```diff
@@ -19,6 +19,8 @@ f36097e7-7d66-4196-8d95-6b66f31435cc,,,See comments,See comments,Commentaires,Co
 2a148983-c143-4649-b4be-f8845c934ec0,,,Nightly,Nightly,Chaque nuit,Chaque nuit,,,Frequency,N/A,,,,,,,,,,,
 ,,CIEL:1065,Yes,Yes,Oui,Oui,,,Misc,N/A,,,,,,,,,,,
 ,,CIEL:1066,No,No,Non,Non,,,Misc,N/A,,,,,,,,,,,
+b821d08c-34f3-4614-9ae6-30a80bce0236,,CIEL:1499,Moderate,Moderate,Modérer,Modérer,,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/misc.csv]: "Modéré"

File: configuration/concepts/nutrition.csv [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+edaef9f4-2b5b-4b71-9019-74f1d40ad4d7,,,Oedema,Œdème,Oedema,Œdème,,,Misc,N/A,,,,,,,,,,,
+1503f027-2693-4674-b658-de41d5df60e4,,,Weight-for-height<-3SD/BMI<16,PT <-3ET (IMC <16),Weight-for-height<-3SD/BMI<16,PT <-3ET (IMC <16),,,Misc,N/A,,,,,,,,,,,
+8dfc3ec9-771a-42b6-ad45-aa0ac15b660c,,,MUAC<115mm,PB<115mm,MUAC<115mm,PB<115mm,,,Misc,N/A,,,,,,,,,,,
+6a7ea608-0546-40f1-92d2-cd020e980414,,,Admission Criteria,Critère d'admission,Admission Criteria,Critère d'admission,,,Misc,Coded,,Oedema;Weight-for-height<-3SD/BMI<16;MUAC<115mm,,,,,,,,,
+a2224476-1827-4270-9ace-c8fd7898fd21,,,Readmission,Réadmission,Readmission,Réadmission,,,Misc,Coded,,Yes;No,,,,,,,,,
+fc1f3f86-9332-4714-a4f4-ce88c173a5c2,,,Weight goal,Objectif de poids,Weight goal,Objectif de poids,,,Misc,Numeric,,,,TRUE,Kg,,,,,,
+5f2ee7bc-8dcf-4617-b31c-2ac6c07617b5,,,0,0,0,0,,,Misc,N/A,,,,,,,,,,,
+e7c0fefc-96e8-4ccb-b24b-92ec2024f579,,CIEL:1362,+,+,+,+,,,Misc,N/A,,,,,,,,,,,
+9f14f988-be10-419b-b947-0508664b7ef2,,CIEL:1363,++,++,++,++,,,Misc,N/A,,,,,,,,,,,
+068dad25-16ff-4468-8f49-fa1666d493bf,,,Grade of edema,Présece d'oedèmes,Grade of edema,Présece d'oedèmes,,,Misc,Coded,,0;+;++,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/nutrition.csv]: "Présence d'œdèmes"

File: configuration/concepts/nutrition.csv [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+edaef9f4-2b5b-4b71-9019-74f1d40ad4d7,,,Oedema,Œdème,Oedema,Œdème,,,Misc,N/A,,,,,,,,,,,
+1503f027-2693-4674-b658-de41d5df60e4,,,Weight-for-height<-3SD/BMI<16,PT <-3ET (IMC <16),Weight-for-height<-3SD/BMI<16,PT <-3ET (IMC <16),,,Misc,N/A,,,,,,,,,,,
+8dfc3ec9-771a-42b6-ad45-aa0ac15b660c,,,MUAC<115mm,PB<115mm,MUAC<115mm,PB<115mm,,,Misc,N/A,,,,,,,,,,,
+6a7ea608-0546-40f1-92d2-cd020e980414,,,Admission Criteria,Critère d'admission,Admission Criteria,Critère d'admission,,,Misc,Coded,,Oedema;Weight-for-height<-3SD/BMI<16;MUAC<115mm,,,,,,,,,
+a2224476-1827-4270-9ace-c8fd7898fd21,,,Readmission,Réadmission,Readmission,Réadmission,,,Misc,Coded,,Yes;No,,,,,,,,,
+fc1f3f86-9332-4714-a4f4-ce88c173a5c2,,,Weight goal,Objectif de poids,Weight goal,Objectif de poids,,,Misc,Numeric,,,,TRUE,Kg,,,,,,
+5f2ee7bc-8dcf-4617-b31c-2ac6c07617b5,,,0,0,0,0,,,Misc,N/A,,,,,,,,,,,
+e7c0fefc-96e8-4ccb-b24b-92ec2024f579,,CIEL:1362,+,+,+,+,,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/nutrition.csv]: Could you set the FSN as "One plus" and "Two plus"
in fr: "Un plus", "Deux plus"
and keep the short name as "+" and "++"

File: configuration/concepts/nutrition.csv [hunk 4]
```diff
@@ -0,0 +1,51 @@
+uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:fr,Short name:en,Short name:fr,Description:en,Description:fr,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:1400
+edaef9f4-2b5b-4b71-9019-74f1d40ad4d7,,,Oedema,Œdème,Oedema,Œdème,,,Misc,N/A,,,,,,,,,,,
+1503f027-2693-4674-b658-de41d5df60e4,,,Weight-for-height<-3SD/BMI<16,PT <-3ET (IMC <16),Weight-for-height<-3SD/BMI<16,PT <-3ET (IMC <16),,,Misc,N/A,,,,,,,,,,,
+8dfc3ec9-771a-42b6-ad45-aa0ac15b660c,,,MUAC<115mm,PB<115mm,MUAC<115mm,PB<115mm,,,Misc,N/A,,,,,,,,,,,
+6a7ea608-0546-40f1-92d2-cd020e980414,,,Admission Criteria,Critère d'admission,Admission Criteria,Critère d'admission,,,Misc,Coded,,Oedema;Weight-for-height<-3SD/BMI<16;MUAC<115mm,,,,,,,,,
+a2224476-1827-4270-9ace-c8fd7898fd21,,,Readmission,Réadmission,Readmission,Réadmission,,,Misc,Coded,,Yes;No,,,,,,,,,
+fc1f3f86-9332-4714-a4f4-ce88c173a5c2,,,Weight goal,Objectif de poids,Weight goal,Objectif de poids,,,Misc,Numeric,,,,TRUE,Kg,,,,,,
+5f2ee7bc-8dcf-4617-b31c-2ac6c07617b5,,,0,0,0,0,,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/nutrition.csv]: Not sure what to do with this one... Probably the FSN should be "Zero" (in fr: Zéro)

File: configuration/concepts/nutrition.csv [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+4f73dc25-e114-4666-8559-e3e2528bac7a,,,Thirst,Soif,Thirst,Soif,,,Misc,Coded,,Yes;No,,,,,,,,,
+f9270e85-db3f-4e64-8f2a-be3b13863708,,,Malnutrition Signs and symptoms,Symptômes de malnutrition,Malnutrition Signs and symptoms,Symptômes de malnutrition,,,Misc,Coded,,Cough (R05);Fever;Diarrhea (> 3 watery stools);Vomiting (R11);Frequent urination;Thirst,,,,,,,,,
+0fdb609d-41d2-401b-9f8f-e96c98aa0bdc,,,Malnutrition Signs and symptoms set,Symptômes de malnutrition set,Malnutrition Signs and symptoms set,Symptômes de malnutrition set,,,Misc,ConvSet,,,Malnutrition Signs and symptoms;Duration;Time Units,,,,,,,,
+90eec1b7-57fc-41c4-9974-6f9587bdf368,,,Chest reinforcement pull,Tirage ou renforcement thoracique,Chest reinforcement pull,Tirage ou renforcement thoracique,,,Misc,Coded,,Yes;No,,,,,,,,,
+f4da9202-f358-469c-9987-55cae483cbbb,,,Signs of dehydration,Signes de déshydratation,Signs of dehydration,Signes de déshydratation,,,Misc,Coded,,No;Moderate;Severe,,,,,,,,,
+910a2ec7-0672-4880-a2b9-1f9938323611,,,Restless,Agité,Restless,Agité,,,Misc,N/A,,,,,,,,,,,
+279561c9-e537-42c7-87cc-c16fe6eb6c6e,,,Irritable,Iritable,Irritable,Iritable,,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/nutrition.csv]: in fr "Irritable" (2 r)

File: configuration/concepts/nutrition.csv [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+3af92575-4fde-4982-ae14-85f815c452c8,,,Passive,Passif,Passive,Passif,,,Misc,N/A,,,,,,,,,,,
+afff0d91-df0f-4913-a34a-22e94628bbc5,,,State of consciousness,Etat de conscience,State of consciousness,Etat de conscience,,,Misc,Coded,,Passive;Restless;Irritable,,,,,,,,,
+24a58167-e111-48a7-953b-d4c169964821,,,Cold,Froides,Cold,Froides,,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/nutrition.csv]: Isn't there a CIEL mapping for that?
In fr, we should probably put "Froid(e)"

File: configuration/concepts/nutrition.csv [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+7ea58c8f-be4c-4c7c-80d7-e43459be45d4,,,Extremities,Extremités,Extremities,Extremités,,,Misc,Coded,,Normal;Cold,,,,,,,,,
+6513d6c6-d706-4419-9311-ea7e32b2c5b1,,,Dry,Sèches,Dry,Sèches,,,Misc,N/A,,,,,,,,,,,
+566d50e2-5041-4b81-b6d0-da87fdbf1c2f,,,Flow,Écoulement,Flow,Écoulement,,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/nutrition.csv]: Isn't there a CIEL mapping for that?

File: configuration/concepts/nutrition.csv [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+c1db62b1-082b-46b1-b2a8-bb8da60a6e5a,,,Neck,Cou,Neck,Cou,,,Misc,N/A,,,,,,,,,,,
+cba34152-fbda-4953-bc5c-0cc44db433a1,,,Inguinal fold,Pli inguinal,Inguinal fold,Pli inguinal,,,Misc,N/A,,,,,,,,,,,
+8d182e71-f10f-42de-b79e-6005917751bb,,,Lymph Nodes,Ganglions,Lymph Nodes,Ganglions,,,Misc,Coded,,Normal;Axillary;Neck;Inguinal fold,,,,,,,,,
+4dc86194-6a6b-44d5-8681-2d6d15f61577,,,Other Problems,Autres problèmes,Other Problems,Autres problèmes,,,Misc,Coded,,Yes;No,,,,,,,,,
+872f2b62-dbe7-430e-ba77-fa1440b170b7,,,Other Problems Details,Détails des autres problèmes,Other Problems Details,Détails des autres problèmes,,,Misc,Text,,,,,,,,,,,
+b82cb558-1b79-4dba-a30a-2f7bd93d8c7c,,,Good,Bon,Good,Bon,,,Misc,N/A,,,,,,,,,,,
+b46aa35e-9fa3-4d61-9d88-e805648c7eac,,,Fair,Moyen,Fair,Moyen,,,Misc,N/A,,,,,,,,,,,
+6c5e5c1d-137c-421c-ae7e-243dd24fbe18,,,Refusal,Refus,Refusal,Refus,,,Misc,N/A,,,,,,,,,,,
+7c383fa5-d85b-4b5f-b41f-d86826403969,,,Appetite,Appétit,Appetite,Appétit,,,Misc,Coded,,Good;Fair;Refusal,,,,,,,,,
+43da897c-9936-40db-9b19-7be549505d93,,,Requires a follow-up appointment,Nécéssite un RV de suivi,Requires a follow-up appointment,Nécéssite un RV de suivi,,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/nutrition.csv]: "Nécéssite un RDV de suivi"

File: configuration/concepts/nutrition.csv [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+713ab6cf-71e6-4f69-9a8a-747b948a5656,,,Cured (Reached weight goal),Guéri (A atteint l'objectif de poids),Cured (Reached weight goal),Guéri (A atteint l'objectif de poids),,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/nutrition.csv]: Guéri (a atteint l'objectif de poids)

File: configuration/concepts/nutrition.csv [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+08e3c76b-f95c-48e2-9249-51311f3453ec,,,Fail (Does not reach weight goal after 12 weeks),Échec (N'atteint pas l'objectif de poids après 12 semaines),Fail (Does not reach weight goal after 12 weeks),Échec (N'atteint pas l'objectif de poids après 12 semaines),,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/nutrition.csv]: "Échec (n'atteint pas l'objectif de poids après 12 semaines)"

File: configuration/concepts/nutrition.csv [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+3cd5bb6d-29ae-4f3c-90e4-dd77fb08c2ca,,,Withdrawal (Does not show up at three consecutive follow-up appointments),Abandon (Ne se présente pas à trois RV de suivi consécutif),Withdrawal (Does not show up at three consecutive follow-up appointments),Abandon (Ne se présente pas à trois RV de suivi consécutif),,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/nutrition.csv]: Abandon (ne se présente pas à trois RDV de suivi consécutif)

**Review by rbuisson — CHANGES_REQUESTED** [review]
See comments
Seems like we could find some more CIEL mappings. Especially for all the signs "Oral thrush", "Inflammation"...

File: configuration/concepts/misc.csv (hunk 1 above)
Comment (reagan-meant) [reply configuration/concepts/misc.csv]: Done

File: configuration/concepts/nutrition.csv (hunk 2 above)
Comment (reagan-meant) [reply configuration/concepts/nutrition.csv]: Done

File: configuration/concepts/nutrition.csv (hunk 3 above)
Comment (reagan-meant) [reply configuration/concepts/nutrition.csv]: Done

File: configuration/concepts/nutrition.csv (hunk 4 above)
Comment (reagan-meant) [reply configuration/concepts/nutrition.csv]: Done

File: configuration/concepts/nutrition.csv (hunk 5 above)
Comment (reagan-meant) [reply configuration/concepts/nutrition.csv]: Done

File: configuration/concepts/nutrition.csv (hunk 8 above)
Comment (reagan-meant) [reply configuration/concepts/nutrition.csv]: Done

File: configuration/concepts/nutrition.csv (hunk 9 above)
Comment (reagan-meant) [reply configuration/concepts/nutrition.csv]: Done

File: configuration/concepts/nutrition.csv (hunk 10 above)
Comment (reagan-meant) [reply configuration/concepts/nutrition.csv]: Done

File: configuration/concepts/nutrition.csv (hunk 11 above)
Comment (reagan-meant) [reply configuration/concepts/nutrition.csv]: Done

**Review by rbuisson — APPROVED** [review]

**Pull request merged**